src/pages/student_fees_list/FeesList.tsx

```diff
@@ -0,0 +1,277 @@
+  const [selectedFeeId, setSelectedFeeId] = useState<string | null>(null);
+  const [selectedPaymentId, setSelectedPaymentId] = useState<string | null>(
+    null
+  );
+  const [payMonthlyFee] = useUpdateMonthlyFeeMutation();
+
+  const normalizeDate = (date: Date | string | null) => {
+    if (!date) return undefined;
+
+    const d = new Date(date);
+
+    if (isNaN(d.getTime())) return undefined;
+    return d.toISOString().split("T")[0]; // "YYYY-MM-DD"
+  };
+  const formattedExpirationDate = normalizeDate(expirationDateFilter);
+  const {
+    data: fees,
+    isLoading,
+    isError,
+  } = useGetStudentMonthlyFeesQuery(
+    student?.id
+      ? {
+          studentId: student.id,
+          month: monthFilter || undefined,
+          expirationDate: formattedExpirationDate,
+          status: statusFilter || undefined,
+        }
+      : skipToken
+  );
+
+  const handleOpenPayModal = (feeId: string) => {
+    setSelectedFeeId(feeId);
+    setModalType("pay");
+  };
+  const handleOpenPaymentInfoModal = (paymentId?: string) => {
+    setSelectedPaymentId(paymentId!);
+    setModalType("details");
+  };
+  const handleConfirmPay = async () => {
+    if (!selectedFeeId) return;
+    payMonthlyFee({
+      feeId: selectedFeeId,
+      studentId: student.id,
+    })
+      .unwrap()
+      .then(() => {
+        toast.success("Pago realizado correctamente");
+      })
+      .catch(() => {
+        toast.error("Ocurrió un error al procesar el pago");
+      })
+      .finally(() => {
+        setModalType(null);
+        setSelectedFeeId(null);
+      });
+  };
+
+  const handleCancelPay = () => {
+    setModalType(null);
+  };
+  if (isLoading) return <p>Cargando cuotas...</p>;
+  if (isError) return <p>Error al cargar cuotas.</p>;
+
+  const columns: Column<StudentMonthlyFeeResponse>[] = [
+    {
+      header: <SortableButton text="N° de cuota" />,
+      accessor: "number",
+    },
+
+    {
+      header: <SortableButton text="Mes" />,
+      accessor: "month",
+      Cell: ({ original }) => <span>{translateMonth(original.month)} </span>,
+    },
+
+    {
+      header: <SortableButton text={"Fecha de\nvencimiento"} allowWrap />,
+      accessor: "expirationDate",
+      Cell: ({ original }) => <span>{original.expirationDate}</span>,
+    },
+    {
+      header: <SortableButton text="Monto" />,
+      accessor: "amount",
+      Cell: ({ original }) => <span>{formatCurrency(original.amount)}</span>,
+    },
+    {
+      header: "Estado",
+      accessor: "status",
+      Cell: ({ original }) => (
+        <FeeStatusContainer>
+          <FeeStatus $status={original.status}>{original.status}</FeeStatus>
+        </FeeStatusContainer>
+      ),
+    },
+    {
+      header: "Pago",
+      Cell: ({ original }) => {
+        const canPay = ["Pendiente", "Vencida"].includes(original.status);
+        const canViewPayment = ["Pagado", "Pagado vencido"].includes(
+          original.status
+        );
```

Comment:
Esto esta muy bien, pero deberias definir los array de status como constantes con nombres que representen que son esos estados en particular, por ejemplo, definimos estas constantes

```
export const MONTHLY_FEE_STATUS_CAN_BE_PAID = ["Pendiente", "Vencida"];
export const MONTHLY_FEE_STATUS_CAN_VIEW_PAYMENT = ["Pagado", "Pagado vencido"];
```
y aca te quedaria de esta manera

```suggestion
        const canPay = MONTHLY_FEE_STATUS_CAN_BE_PAID.includes(original.status);
        const canViewPayment = MONTHLY_FEE_STATUS_CAN_VIEW_PAYMENT.includes(original.status);
```